core/zip/inc/Compression.h

```diff
@@ -31,6 +31,15 @@ namespace ROOT {
 /// The current algorithms support level 1 to 9. The higher the level the greater
 /// the compression and more CPU time and memory resources used during compression.
 /// Level 0 means no compression.
+///
+/// Recommendation for the compression algorithm's levels:
+///  - ZLIB is recommended to be used with compression level 6 [106] 
+///  - LZMA is recommended to be used with compression level 7-8 (higher is better, 
+///   since in the case of LZMA we don't care about compression/decompression speed)
+///   [207 - 208]
+///  - LZ4 is recommended to be used with compression level 5 [405]
```

Comment:
From https://indico.fnal.gov/event/15154 and https://indico.fnal.gov/event/15154/contribution/8/material/slides/0.pdf, it seems we settled for level 4.  However, I currently can't locate the meeting minutes (@dpiparo do you remember if we took some?)

Reply:
I see it on slide 10 from Jim's presentation. He was measuring two groups of levels [1-3] and [4-9]. You  are right, I will fix level = 4 (anyway I will re-measure results  for ROOT I/O workshop, so we can change it in case the results will be different)